lms/views/helpers/_authorization.py

```diff
@@ -0,0 +1,13 @@
+__all__ = ("is_instructor",)
+
+
+def is_instructor(request):
```

Comment:
I've moved this little util function into a helper here out of desperation not to reproduce its logic _for a third time_. This isn't where it belongs ultimately, but I'd at least like to try to avoid the copypasta situation that led to a bug that I found when testing this (`teachingassistant` had been misspelled as `teachingassisstant` in the variant I started from).